references/cli/adopt.go

```diff
@@ -0,0 +1,453 @@
+				return false, err
+			}
+			spinner.UpdateCharSet([]string{fmt.Sprintf("waiting application %s/%s to be running, current status: %s", app.Namespace, app.Name, _app.Status.Phase)})
+			return _app.Status.Phase == common.ApplicationRunning, nil
+		})
+		spinner.Stop()
+		if err != nil {
+			return fmt.Errorf("failed to wait application %s/%s to be running: %w", app.Namespace, app.Name, err)
+		}
+		switch opt.Type {
+		case adoptTypeHelm:
+			for _, r := range opt.HelmReleases {
+				if _, err = opt.HelmStore.Delete(r.Name, r.Version); err != nil {
+					return fmt.Errorf("failed to clean up helm release: %w", err)
+				}
+			}
+			_, _ = fmt.Fprintf(opt.Out, "successfully clean up old helm release\n")
+		default:
+		}
+	}
+	return nil
+}
+
+var (
+	adoptLong = templates.LongDesc(i18n.T(`
+		Adopt resources into applications
+
+		Adopt resources into a KubeVela application. This command is useful when you already
+		have resources applied in your Kubernetes cluster. These resources could be applied
+		natively or with other tools, such as Helm. This command will automatically find out
+		the resources to be adopted and assemble them into a new application.
+
+		There are two types of adoption supported by far, native (by default) and helm.
+		1. For native type, you can specify a list of resources you want to adopt in the
+		application. Only resources in local cluster are supported for now.
+		2. For helm type, you can specify a helm release name. This helm release should
+		be already published in the local cluster. The command will find the resources
+		managed by the helm release and convert them into an adoption application.
+
+		There are two modes for the adoption by far, read-only mode (by default) and 
+		take-over mode.
+		1. In read-only mode, adopted resources will not be touched. You can leverage vela 
+		tools (like Vela CLI or VelaUX) to observe those resources and attach traits to add 
+		new capabilities. The adopted resources will not be recycled or updated. This mode
+		is recommended if you still want to keep using other tools to manage resources updates
+		or deletion, like Helm.
+		2. In take-over mode, adopted resources can be modified. You can use traits or
+		directly modify the component to make edits to those resources. This mode can be 
+		helpful if you want to migrate existing resources into KubeVela system and let KubeVela
+		to handle the life-cycle of target resources.
+
+		The adopted application can be customized. You can provide a CUE template file to
+		the command and make your own assemble rules for the adoption application. You can
+		refer to https://github.com/kubevela/kubevela/blob/master/references/cli/adopt.cue to
+		see the default implementation of adoption rules.
+	`))
+	adoptExample = templates.Examples(i18n.T(`
+		# Native Resources Adoption
+
+		## Adopt resources into new application
+		vela adopt deployment/my-app configmap/my-app
```

Comment:
```suggestion
		vela adopt <API-resources>[/<resource-namespace>]/[resource-name] <API-resources-2>[/<resource-namespace-2>]/[resource-name-2]
```

You should add pattern in the first place to let users know.

Reply:
Done